### DISCUSSION TITLE
Multi-Form Pages related bugs

### DISCUSSION THREAD

**Discussion by FactorD** [team-forms/community#23] [Ideas]
Hi,

I'd like to share two bugs which are quite frustrating. 

#1: When editing pages, on save the system might create several copies of the edited page and sometimes even destroy page order. 

#2: Setting Page#1 to be hidden for further edits of the initial submit doesn't work. The first page is technically hidden, the breadcrumb doesn't show as intended, but when opening the edit link the form will still open on page 1.

**Comment by AnthonyPhan** [comment]
Hi @FactorD,

Thanks for your report. We can confirm that we are aware of the issue with editing pages within a multi-page wizard and plan to fix this in a future release.  I'll use this discussion to post any updates as they become available.

As for the second issue you raise, i'll need to investigate this one further. I think it's worth keeping each discussion to a single issue so that we can easily track and update the discussions. Would you mind raising this as a separate issue or discussion topic? That way others may be able to read on the wizard editing issues without also needing the read though discussions on an unrelated issue.
